### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Local IIS Add Server_Keyboard:After pressing enter on "cancel" button when we are  pressing "tab" focus is moving to top of the page to "Hide Side Bar" control

### ISSUE THREAD

**Issue by Amlan454** [Microsoft/IIS.WebManager#143]
Environment Details:
URL: https://manage.iis.net/settings/servers
Browser: Edge(42.17134.1.0)Chrome(68.0.3440.106)
Keyboard
MAS ID Violated: MAS 2.4.3 

Repro Steps:
1.Open "Microsoft IIS" application using the URL: https://manage.iis.net/
2.Download and install "Microsoft IIS Administration" application.
3.Tab till Local IIS  on the top navigation bar and press enter
5.Tab till add server button and press enter
6.Tab till cancel button and press enter
7.Now press Tab and check whether the focus order is correct or not

Actual Result:
After pressing enter on "cancel" button when we are  pressing "tab" focus is moving to top of the page to "Hide Side Bar" control

Expected Result:
After pressing enter on cancel button when we are pressing tab focus should go to Add Server button

User Impact:
Keyboard only users will face difficulty when navigating through the application

Mas Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

Reference video:
[Bug_143_focus order is incorrect.pptx](https://github.com/Microsoft/IIS.WebManager/files/2330801/Bug_143_focus.order.is.incorrect.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #Keyboard; #MAS2.4.3; #C+E-H; #.NETWebsites; #Chrome; #Edge

**Comment by Yashi1201** [comment]
**Github tags**: #A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
